README.md

```diff
@@ -116,3 +122,9 @@ Several scripts form a pipeline for updating the database. Here is the data flow
                         └──────────────────────┘ │
                           └──────────────────────┘
 ```
+
+> **NOTES**:
+>
+> * These scripts also garbage-collect unreferenced data.
```

Comment:
For me, garbage collection is for memory management. I would use _delete_.

It's stated [there](https://github.com/ghuser-io/db/pull/1/commits/7c6efecdb57c350dfb85046be1f8f152bb2ec2e5#diff-04c6e90faac2675aa89e2176d2eec7d8R45) but I think it's fine to repeat if you think it's important. I think it's not too important because those files are in a repo, so you cannot delete something by accident.

Reply:
> It's stated there

actually I find that "clean" is vague, it could be as well meaning transforming some "dirty" data into "cleaner" data, so applying transformations.

"deleting unreferenced data" is crystal-clear I'd say.